### DISCUSSION TITLE
Comp Bot Gearing

### DISCUSSION THREAD

**Discussion by PChild** [team401/2024-Robot-Code#61] [General]
I made a "[Comp Bot Gearing](https://docs.google.com/document/d/1oZgSedfQl8sXPFJ26hh-nO25nslIkLVV-vtROwnsgb0/edit)" doc in the 2024 Controls folder with some info on gearing and center of mass for most systems.

Is there anything that you all think needs changing?

**Comment by honzikschenk** [comment]
@PChild What's the conversion factor for endgame? I have the gearing but not pitch.

**Reply by PChild** [reply]
It was in the recalc links, I pasted it into the doc. You probably want to check the recalc links for all systems, there are calculated gains that we can use as baselines for our systems.

**Reply by honzikschenk** [reply]
Thanks
